integration/models/README.md

```diff
@@ -0,0 +1,9 @@
+# Model Integration Tests
+
+Populate this directory with a set of model jsonl files you want to test.  These files are ignored by git and
+should not be checked in, but populated dynamically.
+
+```json
+{"name":"llama3","type":"chat"}
```

Comment:
This is awesome. Do you know if we'd be able to make these args one can pass to `go test ./...` vs reading a jsonl file - e.g. `go test -run TestModel llama3 llama3:70b ...`. Not a huge thing, just curious for easier testing w/o editing a file

Reply:
Yes, these can be passed in.  `-run TestModels/llama3:70b`